game_view.cpp

```diff
@@ -408,9 +409,8 @@ void test_game_view() //!OCLINT tests may be many
     }
 
     {
-    const game_view v;
-    // game has a member function called `get_n_ticks`, which returns zero upon construction
-    assert(v.get_game().get_n_ticks() == 0);
+        game_view v;    // game has a member function called `get_n_ticks`, which returns zero upon construction
```

Comment:
I think we could keep "const" there. Is there a good reason why "const" is not used?

Reply:
good point, fixed 👍